config/config/assets/registry-config.yaml.tmpl

```diff
@@ -145,6 +145,16 @@ storage:
     realm: {{ .Spec.Storage.Driver.Azure.BaseURL | quote }}
   {{- end }}
 
+  {{- if .Spec.Storage.Driver.AliOSS }}
+  oss:
```

Comment:
According to docker distribution documentation, a lot more options are available like chunksize, endpoint, encryption options.

Reply:
there would be some confuse when using `endpoint` between registry and chartmuseum, the format are different

Reply:
I still fix this, as it may be useful for user to use internal endpoint